### PR TITLE
Add markdownz to editable fields

### PR DESCRIPTION
**Pull request by simoneduca**
List of fields that need markdown support

- [x] all project pages
- [x] tutorial (step only)
- [x] Field guide (content only)

### CODE REVIEW

File: src/containers/ProjectContentsContainer.jsx [hunk 1]
```diff
@@ -113,8 +113,15 @@ class ProjectContentsContainer extends Component {
           <ModalBody>
             <h2>Original</h2>
             <p>{this.state.fieldText}</p>
-            <h2>Translation</h2>
-            <input type="text" onChange={this.handleChange} autoFocus placeholder="Translate some text" />
+            <h2>Your translation</h2>
+            <MarkdownEditor
```
Comment (eatyourgreens) [inline src/containers/ProjectContentsContainer.jsx]: This should be a textarea or text input if the field is plain text.

You should add a flag to fields that need the markdown editor, eg. `data-markdown=true` then check the flag here (`event.target.getAttribute('data-markdown')` and render either a plain text input or the markdown editor.

File: src/styles/components/markdown-editor-preview.styl [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.markdown-editor-preview
+  display none
```
Comment (eatyourgreens) [inline src/styles/components/markdown-editor-preview.styl]: Do you think people will want to use the preview button to look at their changes? We can leave that until later but worth keeping in mind.

File: src/containers/ProjectContentsContainer.jsx (hunk 1 above)
Comment (simoneduca) [inline src/containers/ProjectContentsContainer.jsx]: Agh, my bad sorry.

File: src/styles/components/markdown-editor-preview.styl (hunk 2 above)
Comment (simoneduca) [inline src/styles/components/markdown-editor-preview.styl]: Having a preview would be nice I think. But I'm not sure it works properly. The property `previewing` is false by the default, so I don't understand why it was showing anyway, without this css.

Comment (eatyourgreens) [inline src/styles/components/markdown-editor-preview.styl]: I'm happy to leave this and figure it out later.

At worst, we could leave the preview showing under the editor, but only if it updates when you change the markdown.

**Comment by simoneduca** [comment]
Should be fixed now. Also I realised the markdown viewer I added to field guide, tutorials and project pages wasn't in the right place so I removed it.

**Comment by eatyourgreens** [comment]
👍🏻 A couple of small comments. Nothing major.

**Pull request merged**